File: app/src/main/java/han/ayeon/searchimgandvideo/model/web/SearchService.java
package han.ayeon.searchimgandvideo.model.web;

import han.ayeon.searchimgandvideo.model.data.FetchMediaApiResult;
import io.reactivex.disposables.Disposable;

public interface SearchService {
    void search(String searchWord, FetchMediaApiResult queryCallBack);
}

File: app/src/main/java/han/ayeon/searchimgandvideo/model/web/SearchService.kt
package han.ayeon.searchimgandvideo.model.web

import han.ayeon.searchimgandvideo.model.data.FetchMediaApiResult

/**
 * Created by HanAYeon on 2019-07-17.
 */

interface SearchService {

    fun search(searchWord : String, queryCallBack : FetchMediaApiResult)
}